integration/pxe/README.md

```diff
@@ -1,28 +1,76 @@
 # PXE
 
-This a DHCP, TFTP and HTTP servers
+This a container with DHCP, TFTP and HTTP servers
 
-Mount points:
+## Mount points
 
 - /var/lib/tftpboot/ serves (initrd, kernel, grub.cfg)
 - /etc/dhcp/dhcpd.conf config file for DHCP
 
-Environment variables:
+## Environment variables
 
 - NODE_IP_SUBNET - The DHCP subnet
 - NODE_IP_NETMASK - The DHCP netmask
 - NODE_IP_RANGE_MIN - The DHCP Min address
 - NODE_IP_RANGE_MAX - The DHCP Max address
 - NODE_IP_ADDRESS - The current node IP address
 
-Run Example:
+## Run
 
 ```text
 docker-compose -f docker-compose.pxe.yml up --build
 ```
 
-Test:
+## Attach to any container on the same network
```

Comment:
@Boris are you asking user to attach to any other container on the same network as this container, which was just built, and then to run the following commands expecting the output similar to the output below?
I was a bit confused. If so then this looks good to me with maybe a heading that is more explicit.

Reply:
yep, from any container on the same network you can discover the DHCP and HTTP and TFTP (PXE)
manual ask here is just an example
eventually we will run ZTP or PZE agent in one of the existing containers automatically to test provisioning